cmd/mattermost/commands/command_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package commands
+
+import (
+	"testing"
+
+	"github.com/mattermost/mattermost-server/api4"
+	"github.com/mattermost/mattermost-server/model"
+)
+
+func TestDeleteCommand(t *testing.T) {
+	th := api4.Setup().InitBasic()
+	defer th.TearDown()
+	url := "http://localhost:8000/test-command"
+	team := th.BasicTeam
+	user := th.BasicUser
+	th.LinkUserToTeam(user, team)
+
+	// Check the appropriate permissions are enforced.
+	defaultRolePermissions := th.SaveDefaultRolePermissions()
+	defer func() {
+		th.RestoreDefaultRolePermissions(defaultRolePermissions)
+	}()
+	id := model.NewId()
+	c := &model.Command{
+		DisplayName: "dn_" + id,
+		Method:      "G",
+		TeamId:      team.Id,
+		Username:    user.Username,
+		URL:         url,
+		Trigger:     "test",
+	}
+	th.AddPermissionToRole(model.PERMISSION_MANAGE_SLASH_COMMANDS.Id, model.TEAM_USER_ROLE_ID)
+	command, _ := th.Client.CreateCommand(c)
+	CheckCommand(t, "command", "delete", command.Id)
```

Comment:
Would be nice to check if it is actually deleted or no after the command.

Would be nice to have a not-valid command id deletion to see if it fails correctly.

Reply:
I added the test but I see that this tests on this file keep failing on my machine even on master without my change. So I am not able to verify if my test works.